charts/lagoon-core/templates/api.ingress.yaml

```diff
@@ -12,6 +12,9 @@ metadata:
     {{- toYaml . | nindent 4 }}
   {{- end }}
 spec:
+  {{- if or (.Values.api.ingress.ingressClassName) (.Values.defaultIngressClass) }}
+  ingressClassName: {{coalesce .Values.api.ingress.ingressClassName .Values.defaultIngressClass }}
+  {{- end }}
```

Comment:
It might be clearer to make this logic look like this:
```
  {{- with coalesce .Values.api.ingress.ingressClassName .Values.defaultIngressClass }}
  ingressClassName: {{ . }}
  {{- end }}
```

Reply:
The downside of this is b/c - if there is no ingressClassName or defaultIngressClassName specified, it outputs a null field on the Ingress - which may be undesirable?